src/controllers/subscription.js

```diff
@@ -1,69 +1,29 @@
-const express = require('express');
-const fs = require('fs');
-const subscriptions = require('../data/subscription.json');
+const Subscription = require('../models/Subscription');
 
-const router = express.Router();
+const createSubs = (req, res) => {
+  Subscription.create(
+    req.body,
+  ).then((result) => res.status(201).json(result))
```

Comment:
**_suggestion_**:
Not matter if a response is correct or wrong it must follow the given format named on this week problematic. Just like you did when an error ocurred (line 10), when the subscription is succesfully created you could use:

.then((result) => res.status(201).json({
message: 'Subscription created',
data: result,
error: false,
}))

![image](https://github.com/BaSP-m2023/baru-megarocket-server/assets/102418096/ecb8a43a-f706-40e2-a36a-776b798dcec4)